manager/integration/tests/test_basic.py

```diff
@@ -379,3 +385,12 @@ def test_volume_multinode(clients, volume_name):  # NOQA
 
     volumes = get_random_client(clients).list_volume()
     assert len(volumes) == 0
+
+
+@pytest.fixture
```

Comment:
It's a normal function, not a pytest fixture.

See pytest fixture doc here: https://docs.pytest.org/en/latest/fixture.html